code/go/pkg/validator/validator.go

```diff
@@ -0,0 +1,13 @@
+package validator
+
+import (
+	"github.com/elastic/package-spec/code/go/internal/validator"
```

Comment:
I'm not quite convinced about the import path as it contains many package levels. Ideally it would be `github.com/elastic/package-spec/validator`, but I assume that `code/` will contain implementation also for JS.

Fine for me!

Reply:
++ I'd love for it to be simpler too but I don't know how to make that happen. Indeed the reason was to allow for other language libraries to be added in the future (I copied the structure from ECS: https://github.com/elastic/ecs/tree/master/code).

Reply:
Just thinking loud: you can leave specification files in package-spec and depend on them in implementations: go-package-validator, js-package-validator. But I believe it might be an overkill here as it's relatively small.

As I said, I'm happy to leave it as is or add a single facade file on repo root (so all imports would be: `github.com/elastic/package-spec`).

Reply:
> Just thinking loud: you can leave specification files in package-spec and depend on them in implementations: go-package-validator, js-package-validator. But I believe it might be an overkill here as it's relatively small.

You mean pull out the libraries into their own repos? Or something else?

> As I said, I'm happy to leave it as is or add a single facade file on repo root (so all imports would be: `github.com/elastic/package-spec`).

This could work but I'm not sure what this means for other languages. I think maybe this is something we can revisit once we add JS validation too. I know it would mean updating existing clients at that point but hopefully that's okay since adding a facade would be a backwards-compatible change (at least for some period of time).